README.md

```diff
@@ -151,7 +151,7 @@ and call the appropriate build tool. For example:
 
 We've hacked together a bash script that will iterate over all of the Clojure
 projects and will set their versions to the same version, including the
-versions of the iPlant developed dependencies (aside from metadactyl).
```

Comment:
Pretty sure set-versions works with metadactyl/apps, so this is out of date. Add it to the list I guess, heh.